src/lints/non_exhaustive_struct_added.ron

```diff
@@ -0,0 +1,62 @@
+SemverQuery(
+    id: "non_exhaustive_struct_added",
+    human_readable_name: "#[non_exhaustive] pub struct added",
+    description: "A new #[non_exhaustive] pub struct was added. It cannot be constructed using a struct literal outside of its defining crate, and pattern matching will require a wildcard.",
```

Comment:
I think we usually try to avoid Rust syntax in these fields. Let's play it safe.
```suggestion
    human_readable_name: "non-exhaustive pub struct added",
    description: "A new non-exhaustive pub struct was added. It cannot be constructed using a struct literal outside of its defining crate, and pattern matching will require a wildcard.",
```